yw_website/README.md

```diff
@@ -1,5 +1,6 @@
 # Prerequisites
 In order to run website make sure you have installed:
+* Java (JRE) version 1.7 or 1.8
```

Comment:
java 10 or greater

Reply:
also this is not required to run the website, just to execute the jar locally

ideally the website is on a dedicated server